Calibration/EcalCalibAlgos/src/classes.h

```diff
@@ -0,0 +1,21 @@
+#ifndef Calibration_EcalCalibAlgos_classes_h
+#define Calibration_EcalCalibAlgos_classes_h
+
+#include "DataFormats/Common/interface/Wrapper.h"
+
+#include "Calibration/EcalCalibAlgos/interface/EcalPhiSymRecHit.h"
+#include "Calibration/EcalCalibAlgos/interface/EcalPhiSymInfo.h"
+
+namespace {
+  struct dictionary {
+    EcalPhiSymRecHit dummy11;
+    std::vector<EcalPhiSymRecHit> dummy12;
+    edm::Wrapper<EcalPhiSymRecHitCollection> dummy13;
+
+    EcalPhiSymInfo dummy21;
+    std::vector<EcalPhiSymInfo> dummy22;
+    edm::Wrapper<EcalPhiSymInfo> dummy23;
+  };
+}  // namespace
```

Comment:
```suggestion
```
This `struct` is no longer needed (for long time).

Reply:
The intent at the moment is to have them transient. I will mark them as transient for now. If things should change in the future, we will move them to the DataFormat package.

> > > It is unclear if any thread safety should be implemented using a edm::SerialTaskQueue. The documentation available [here](https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedFrameworkGlobalModuleInterface#edm_one_WatchLuminosityBlocks) does not reflect the current interface of the class.
> > 
> > 
> > Maybe @makortel could take this one
> 
> We're talking specifically all the `mutable`s in `Calibration/EcalCalibAlgos/plugins/EcalPhiSymRecHitProducers.cc`, right? I'll need some time to digest, but in a first glance all of them are potentially unsafe. `SerialTaskQueue` is unlikely the right answer.
> 
> Our of curiosity, where did you find the link https://twiki.cern.ch/twiki/bin/view/CMSPublic/FWMultithreadedFrameworkGlobalModuleInterface#edm_one_WatchLuminosityBlocks? It points to functionality `WatchLuminosityBlocks` that does not exist for `edm::global` modules, but only for `edm::one` modules. I'm just wondering if some other piece of documentation would need to be improved?

I just googled it. I think in general the explanation on when and how to use one/global module could be improved